public/bitemp_graph_helper.js

```diff
@@ -113,6 +114,26 @@ function deleteDoc(uri) {
   });
 }
 
+function changeTextInGraph(chart, params) {
+  var docProp = $('input[name = documentProperty]').val();
+  if(docProp === '') {
+    window.alert('Please enter a document property.');
+  }
+  else {
+    var chart = barChart()
```

Comment:
This block of code is nearly identical to code in getBarChart(). Refactor the common code into a new function.